app/(candidate)/onboarding/[slug]/[step]/components/districts/DistrictPicker.js

```diff
@@ -0,0 +1,58 @@
+'use client'
+import React, { useState } from 'react'
+import BlackButtonClient from '@shared/buttons/BlackButtonClient'
+import DistrictTypeAutocomplete from './DistrictTypeAutocomplete'
+import DistrictNameAutocomplete from './DistrictNameAutocomplete'
+
+export default function DistrictPicker({
+  state,
+  electionYear,        // number, derived from electionDate
+  buttonText = 'Save',
+  onSubmit,
```

Comment:
```suggestion
  onSubmit = async () => Promise.resolve(),
```

that would allow you to clean up the `try` below..